### PR TITLE
iocage readme changes

### PR DESCRIPTION
**Pull request by jungle-boogie**
first pass of changes for readme.

### CODE REVIEW

**Comment by skarekrow** [comment]
Hello!

Thanks for the PR, but a few things aren't going to be merged from it. First, the port will be named `iocage`, I was waiting for @kmoore134 to make a port for me. @bartekrutkowski was just incredibly quick on his feet (appreciate that robak!) and already made one. But I don't expect that to stay as I don't want that naming scheme for iocage. The current `iocage` will become `iocage_legacy` and this will become `iocage` with `iocage_legacy` conflicting with it.

For the final edit, why remove 9.3? Even if it is EOL, I'm stating that 9.3 and up is required, and that remains true whether or not upstream supports it. Many users may stick with it for some time.

I *will* however take the Python 2.7 requirement, d'oh! Good catch. If you could update and then squash this PR with just that one addition, I'll merge this :)

**Comment by bartekrutkowski** [comment]
@skarekrow Regarding the port name, I am afraid software name doesn't get a priority over the port name. New iocage version is not only based on Python, what requires `py-` prefix in Ports Tree, but also has 'incompatible' version number (0.9 currently is lower than `sysutils/iocage` 1.7.6 version number) with existing iocage port, so I am afraid you will have to live with `sysutils/py-iocage` in Ports Tree. I've already updated `sysutils/iocage` port to use the `iocage_legacy` repository.

File: README.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ iocage is a jail/container manager amalgamating some of the best features and
 technologies the FreeBSD operating system has to offer. It is geared for ease
  of use with a simple and easy to understand command syntax.
 
-iocage is in the FreeBSD ports tree as sysutils/iocage.
-To install using binary packages, simply run: `pkg install iocage`
+iocage is in the FreeBSD ports tree as sysutils/py-iocage.
+To install using binary packages, simply run: `sysutils/py-iocage`
```
Comment (bartekrutkowski) [inline README.md:11]: This line is incorrect, it should say `pkg install py27-iocage`.

Comment (jungle-boogie) [reply README.md:11]: Hi @bartekrutkowski,

But it looks like what's in my diff is correct:
https://www.freshports.org/sysutils/py-iocage/

**Comment by jungle-boogie** [comment]
Hi @skarekrow,

I'll work with @bartekrutkowski and determine the correct port/package name and open a new PR then.

Thanks!

File: README.md (hunk 1 above)
Comment (skarekrow) [reply README.md:11]: @jungle-boogie If you look at PKGNAME you will see it's py27-iocage.

@bartekrutkowski We still need a conflict on these, I don't intend for them to coexist.

Comment (jungle-boogie) [reply README.md:11]: @skarekrow,

Yeah, what's up with that. https://www.freshports.org/sysutils/py-iocage but says pkg install py27-iocage

**doesn't exist** https://svnweb.freebsd.org/ports/head/sysutils/py27-iocage 
**exists** https://svnweb.freebsd.org/ports/head/sysutils/py-iocage

Comment (skarekrow) [reply README.md:11]: Yeah all python ports do that so we can have Python 2 and 3 ports coexist. It's a fair trade off. I just committed and gave you credit in title. Thanks again!

**Pull request closed without merging**